syslog-ng: fix wrong ownership issue

Upstream-Status: Pending

The ownership of build user is preserved for some target files, fixed it by
adding --no-same-owner option to tar when extracting files. 

Signed-off-by: Ming Liu <ming.liu@windriver.com>
---
 Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -urpN a/scl/Makefile.am b/scl/Makefile.am
--- a/scl/Makefile.am
+++ b/scl/Makefile.am
@@ -13,7 +13,7 @@ install-data-local:
 		fi; \
 	done
 	$(mkinstalldirs) $(DESTDIR)/$(scldir)
-	(cd $(srcdir); tar cf - $(SCL_SUBDIRS)) | (cd $(DESTDIR)/$(scldir) && tar xf -)
+	(cd $(srcdir); tar cf - $(SCL_SUBDIRS)) | (cd $(DESTDIR)/$(scldir) && tar xf - --no-same-owner)
 
 uninstall-local:
 	for cfg in $(SCL_CONFIGS); do \
